tests/glade_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# If $top_srcdir has not been set by automake, import the test environment
+if [ -z "$top_srcdir" ]; then
+    top_srcdir="$(realpath $(dirname "$0")/..)"
+    . ${top_srcdir}/tests/testenv.sh
+fi
+
+# If no tests were selected by user or makefile, select all of them
+if [ $# -eq 0 ]; then
+    set -- "$top_srcdir"/tests/glade_tests
+fi
+
+exec python3 -m unittest discover -t $top_srcdir -v -b ${UNIT_TESTS_PATTERN:+-k $UNIT_TESTS_PATTERN} -s $@
```

Comment:
I wonder why glade tests are not part of `unit_tests`.

Reply:
Originally I *think* the reason was the plugin, it was written pretty dirty and assumed everything is a glade test. They could be now integrated back, but I wonder how good idea that is. The thing being tested is a fundamentally different one.